tests/test_integration.py

```diff
@@ -632,6 +632,16 @@ def test_self_is_not_installed(projdir, flag, usedevelop):
     assert 'test==0.0.0' not in result.stdout
     assert 'test @ file://' not in result.stdout
 
+@pytest.mark.parametrize("externals", [None, "allowlist_externals", "whitelist_externals"])
+def test_externals(projdir, externals):
+    with modify_config(projdir / 'tox.ini') as config:
+        if externals:
+            config['testenv'][externals] = "foo"
+    result = tox("-e", NATIVE_TOXENV, "--current-env", quiet=False)
+    assert result.returncode == 0
+    assert all(
+        "test command found but not installed in testenv" not in x
+        for x in (result.stdout, result.stderr))
```

Comment:
```suggestion
    assert "test command found but not installed in testenv" not in (result.stdout + result.stderr)
```

Reply:
The thing is, our test command is `python ...`, so we will never get this error line. what about changing it to something else in `modify_config`? Maybe `echo`?